src/openrct2/localisation/StringIds.h

```diff
@@ -3905,9 +3905,10 @@ enum
 
     STR_CREATE_DUCKS_TIP = 6356,
     STR_REMOVE_DUCKS_TIP = 6357,
+    STR_TILE_NEEDS_ATLEAST_ONE_ELEMENT = 6358,
 
     // Have to include resource strings (from scenarios and objects) for the time being now that language is partially working
-    STR_COUNT = 32768
+    STR_COUNT = 32769
```

Comment:
This constant should not be changed.
```suggestion
    STR_COUNT = 32768
```